packages/ui/src/theme/css/component/breadcrumbs.scss

```diff
@@ -0,0 +1,37 @@
+// No styles on .amplify-breadcrumbs
+
+.amplify-breadcrumbs__list {
```

Comment:
Probably people will want to do "something" on narrow layouts with their breadcrumbs, but do you think we should have a default experience of wrapping this list (at the minimum a flex-wrap: wrap, but with a token for theming) so it can collapse on narrow viewports? Currently it will cause a horizontal page scroll.